client/src/components/Button/index.style.js

```diff
@@ -1,6 +1,6 @@
 import styled from "styled-components";
 
-export const StyledButton = styled.button`
+export default styled.button`
```

Comment:
This is inconsistent with the other style, for example `export const StyledDatePicker`. Please use this or the default syntax throughout.

Reply:
I replace this because of the eslint, so if there's one variable exported, it's better to use default (as eslint says)